integration/test_filesystems_integration.py

```diff
@@ -672,3 +673,10 @@ def test_get_path_utils():
     schema, path = get_remaining_path(url)
     assert path == "mycontainer/path/to/object.csv"
     assert schema == "wasbs"
+
+
+def test_ds_get_path_utils():
```

Comment:
I think this is basically a unit test, so better move it there.

Reply:
Hmm, you probably right , and test_get_path_utils() should be moved as well. But there is no unit tests for the utils.py module. Where would you put one?

Reply:
Feel free to create a `test_utils.py` file under `tests/` and move anything that's appropriate to move in there.